src/sasjs-help/index.js

```diff
@@ -170,7 +161,7 @@ export async function printHelpText() {
     )} - Lets the user run a SAS job against a specified target.
     The target can exist either in the local project configuration or in the global .sasjsrc file.
     <sasProgramPath> - It should be the full path to the service, eg that contained in the _program parameter
-    It can also be relative path, for example if appLoc is '/Public/app', then you can pass 'folder/sasjob' it will  
+    It can also be relative path, for example if appLoc is '/Public/app', then you can pass 'folder/sasjob' it will
```

Comment:
Might be worth mentioning that relative paths won't work without an `appLoc` being set.